tools/docker/tests/test_flows.sh

```diff
@@ -137,8 +137,26 @@ test_client_association() {
 }
 
 test_higher_layer_data_payload_trigger() {
-    #TODO: Implement
-    return 1
+    status "test higher layer data payload"
+    
+    mac_gateway_hex=`echo $mac_gateway | tr --delete :`
+    mac_gateway_hex="0x${mac_gateway_hex}"
```

Comment:
We're doing this (converting from colon-MAC to 0x) in several places now, it would make sense to make a function (in functions.sh) for it. (nitpick)

Reply:
you are right, I'll do it :)

Reply:
#282